holmscan/scan.py

```diff
@@ -16,73 +15,80 @@ class Scan(HolmscanModule):
     def __init__(self, controller):
         super(Scan, self).__init__(controller)
 
-    @login
-    def get_web_scan(self, id):
-        url = u"{0}scan/wruns/{1}".format(self.controller.conf.get("HOLMSEC_URL"), id)
-
-        response = self.controller.session.get(url).text
-
-        return response
-
-    @login
-    def start_web_scan(self, **kwargs):
-        data = kwargs
-        data["assets"] = kwargs["assets"]
-        data.update(
-            {
-                "appliance": "-1",
-                "source": 0,
-                "node": "",
-                "user": "{0}".format(self.user_id),
-                "start_date": None,
-            }
-        )
-
-        job_url = u"{0}scan/wscans/".format(self.controller.conf.get("HOLMSEC_URL"))
-        job_response = self.controller.session.post(
-            job_url,
-            data=json.dumps(data).replace(" ", ""),
-            headers={"Content-Type": "application/json"},
-        )
+    def get_net_assets(self):
+        url = "{}/net-scans/assets".format(self.controller.conf.get("HOLMSEC_ENDPOINT"))
```

Comment:
All `{}` formats should use positional arguments saying what item it should use. `{0}, {1} etc`